app/public/index.html

```diff
@@ -16,7 +16,20 @@
 
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
 
-    <link rel="shortcut icon" type="image/png" href="%PUBLIC_URL%/assets/icon/favicon.png" />
+    <!--Favicons-->
+    <link rel="apple-touch-icon" sizes="57x57" href="https://cdn.adam-rms.com/img/icons/apple-icon-57x57.png">
```

Comment:
Can we generate these at compile time?  Using a CDN won't work for devices offline so we should use a manifest.json

Reply:
Yeah, was something I was looking at doing closer to initial release and probably shouldn’t have changed so early

Reply:
Converted to local images, but will need a larger one. Couldn't find one on current app, needs min 144x144px

Reply:
https://github.com/bstudios/adam-rms-app/blob/master/www/assets/logo.png
https://github.com/bstudios/adam-rms-app/blob/master/www/assets/logoicon.svg
https://github.com/bstudios/adam-rms-app/blob/master/www/assets/logo.svg

Reply:
Have replaced with one of those, and is much happier 😄